contrib/examples/actions/echochain.meta.yaml

```diff
@@ -10,4 +10,11 @@ runner_type: "action-chain"
 entry_point: "chains/echochain.yaml"
 
 enabled: true
-parameters: {}
+parameters:
+  skip_notify:
```

Comment:
It did this as long as I can remember :P

In any case, as I noted above - CLI is just one type of our API consumer (others are programming language clients, webui, etc.). We should not bias a general purpose API towards a single consumer.